language-server/modules/langserver-core/src/test/resources/completion/expression_context/config/mapping_expr_ctx_config6.json

```diff
@@ -4,6 +4,7 @@
     "character": 9
   },
   "source": "expression_context/source/mapping_expr_ctx_source6.bal",
```

Comment:
Same as previous one, we should prioritize string variables and functions that return strings.

Reply:
https://github.com/ballerina-platform/ballerina-lang/issues/39408